Mathlib/GroupTheory/Subgroup/Basic.lean

```diff
@@ -2764,6 +2764,11 @@ theorem ofInjective_apply {f : G →* N} (hf : Function.Injective f) {x : G} :
 #align monoid_hom.of_injective_apply MonoidHom.ofInjective_apply
 #align add_monoid_hom.of_injective_apply AddMonoidHom.ofInjective_apply
 
+@[to_additive (attr := simp)]
+theorem apply_ofInjective_symm {f : G →* N} (hf : Function.Injective f) (x : f.range) :
+    f ((ofInjective hf).symm x) = x := by
+  rw [← ofInjective_apply hf, MulEquiv.apply_symm_apply]
```

Comment:
Optional golf:
```suggestion
    f ((ofInjective hf).symm x) = x := 
  Subtype.ext_iff.1 <| (ofInjective hf).apply_symm_apply x
```
to match the proof for `Equiv.apply_ofInjective_symm`

bors d+